src/olympia/reviewers/migrations/0024_backfill-human-review-date-more-agressively.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.18 on 2023-02-21 11:42
+
+from django.db import migrations
+from olympia.constants.base import ADDON_EXTENSION, STATUS_AWAITING_REVIEW
+from olympia.constants.activity import LOG
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('reviewers', '0023_delete_reviewerscore'),
+        ('versions', '0034_auto_20230131_1821'),  # Need human_review_date.
+        ('addons', '0001_initial'),  # Need addons
+        ('files', '0001_initial'),  # Need files
+        ('activity', '0001_initial'),  # Need activity
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                (
+                    """UPDATE `versions`
+INNER JOIN `addons` ON `versions`.`addon_id` = `addons`.`id`
+INNER JOIN `files` ON `versions`.`id` = `files`.`version_id`
+INNER JOIN `log_activity_version`
+  ON `versions`.`id` = `log_activity_version`.`version_id`
+INNER JOIN `log_activity`
+  ON `log_activity_version`.`activity_log_id` = `log_activity`.`id`
+SET `human_review_date`=`log_activity`.`created`
+WHERE `addons`.`addontype_id` = %s
+AND `files`.`status` = %s
+AND `versions`.`human_review_date` IS NULL
+AND `log_activity`.`action` IN (%s, %s)""",
+                    [
+                        ADDON_EXTENSION,
+                        STATUS_AWAITING_REVIEW,
```

Comment:
Not `STATUS_DISABLED`?

Reply:
Aaaand that's why we do reviews ! Definitely meant `STATUS_DISABLED` here.